source/manual/aws-cli-access.html.md

```diff
@@ -0,0 +1,57 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Use AWS on the command line
+section: AWS accounts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-10-10
+review_in: 3 months
+---
+
+💡 Before you can access a thing in AWS, you [need to set up your AWS account](/manual/set-up-aws-account.html).
+
+---
+
+You will need 3 pieces of information:
+
+1. "Role ARN": Find the [Role ARN in the docs][secret-docs]
+1. Your "MFA ARN": the ARN assigned to the MFA device in your account
+1. Access key id and Secret access key
+
+## 1. Create `~/.aws/config`
+
+Create `~/.aws/config`:
+
+```ini
+[profile govuk-<environment>]
+role_arn = <Role ARN>
+mfa_serial = <MFA ARN>
+source_profile = gds
+region = eu-west-1
+
+[profile gds]
+mfa_serial = <MFA ARN>
+region = eu-west-1
+```
```

Comment:
For a minimal config, you can remove `mfa_serial` from the `[profile gds]` section, and `region` from the `[profile govuk-<environment>]` section.

Reply:
👍 awesome